src/api-service/__app__/timer_proxy/__init__.py

```diff
@@ -34,6 +41,17 @@ def main(mytimer: func.TimerRequest, dashboard: func.Out[str]) -> None:  # noqa:
             )
             process_state_updates(proxy)
 
+        if proxy.is_outdated():
+            proxy.outdated = True
```

Comment:
In the future, should `outdated` really be a `state`, instead of an additional field? So we'd have a `ProxyState` that would be an extension of `VmState`.